static/src/javascripts/boot.js

```diff
@@ -45,7 +46,8 @@ const go = () => {
         // keep this in sync with CONSENT_TIMING in src/web/components/App.tsx in frontend
         // mark: CONSENT_TIMING
         let recordedConsentTime = false;
-        onConsentChange(() => {
+        onConsentChange((consentState) => {
```

Comment:
Do we want to send consent data to ophan once per page view or every time consent state changes (if the user changes privacy settings for example)?
If once per page view is enough, it might be worth considering using [getEnhancedConsent](https://github.com/guardian/frontend/blob/df889f77fbab45a89464774882ba9fde80cb4de5/static/src/javascripts/projects/common/modules/commercial/enhanced-consent.ts#L56). This would simplify some of the logic in `decideConsentCarrierLabels` as `getEnhancedConsent` does some of unpacking of `consentState` for you.
We could also remove `recordedConsentTime` below because `getEnhancedConsent` returns a promise of the _first_ user-provided consent state, rather than executing a callback every time consent changes as `onConsentChange` does.